sources/packages/api/src/utilities/student-utils.ts

```diff
@@ -0,0 +1,18 @@
+import { StudentPDStatus } from "../types/pdStatus";
+import { Student } from "../database/entities";
+
+/**
+ * Determines the permanent disablility status from Student Entity.
+ * @param student
+ * @returns PDStatus enum
+ */
+export const determinePDStatus = (student: Student): StudentPDStatus => {
+  if (student.studentPDVerified) {
+    return StudentPDStatus.Yes;
+  } else if (student.studentPDSentAt) {
```

Comment:
I would recommend removing the else here because the return will already interrupt the method execution.
In general, for big methods or even small ones, to reduce the complexity (the same validated on SonarCloud) we can use the "Return Early" pattern. It implies the code and makes it easy to ready.
More info: https://medium.com/swlh/return-early-pattern-3d18a41bba8

@ann-aot @guru-aot, any thoughts?

Reply:
Yes I agree with you @andrewsignori-aot